table.ts

```diff
@@ -140,25 +140,25 @@ export class SabrTable<T> {
   }
 
   /** Checks if a document exists. */
-  has(id: string) {
+  async has(id: string) {
     try {
-      Deno.readFileSync(`${this.sabr.directoryPath}${this.name}/${id}.json`);
+      await Deno.readFile(`${this.sabr.directoryPath}${this.name}/${id}.json`);
       return true;
     } catch {
       return false;
     }
   }
 
   /** Creates a new document into a table. */
-  create(id: string, data: Partial<T> = {}) {
-    if (this.has(id)) {
+  async create(id: string, data: Partial<T> = {}) {
```

Comment:
```suggestion
  async create(id: string, data: T = {}) {
```

would this b better?